apps/marsha/templates/services/app/cronjob_check_live_state.yml.j2

```diff
@@ -0,0 +1,87 @@
+  schedule: "*/20 * * * *"
+  successfulJobsHistoryLimit: 2
+  failedJobsHistoryLimit: 1
+  concurrencyPolicy: Forbid
+  suspend: {{ suspend_cronjob | default(false) }}
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          name: "check-live-state-{{ deployment_stamp }}"
+          labels:
+            app: marsha
+            service: app
+            version: "{{ marsha_image_tag }}"
+            deployment_stamp: "{{ deployment_stamp }}"
+        spec:
+          containers:
+            - name: marsha
+              image: "{{ marsha_image_name }}:{{ marsha_image_tag }}"
+              imagePullPolicy: Always
+              command:
+                - "/bin/bash"
+                - "-c"
+                - python manage.py check_live_state
+              env:
+                - name: DJANGO_SETTINGS_MODULE
+                  value: marsha.configs.settings
+                - name: DJANGO_CONFIGURATION
+                  value: "{{ marsha_django_configuration }}"
+                - name: POSTGRES_DB
+                  value: "{{ marsha_postgresql_database }}"
+                - name: POSTGRES_HOST
+                  value: "marsha-{{ marsha_postgresql_host }}-{{ deployment_stamp }}"
+                - name: POSTGRES_PORT
+                  value: "{{ marsha_postgresql_port }}"
+                - name: DJANGO_ALLOWED_HOSTS
+                  value: "{{ marsha_host | blue_green_hosts }}"
+                - name: DJANGO_CLOUDFRONT_PRIVATE_KEY_PATH
+                  value: "{{ marsha_cloudfront_private_key_path }}"
+              envFrom:
+                - secretRef:
+                    name: "{{ marsha_secret_name }}"
+                - configMapRef:
+                    name: "marsha-app-dotenv-{{ deployment_stamp }}"
+              volumeMounts:
+                - name: marsha-configmap
+                  mountPath: /app/src/backend/marsha/configs
+{% if env_type in trashable_env_types %}
+                - name: marsha-v-media
+                  mountPath: /data/media
+                - name: marsha-v-static
+                  mountPath: /data/static
+{% endif %}
+{% if marsha_should_sign_requests %}
+                - mountPath: "{{ marsha_cloudfront_private_key_path | dirname }}"
+                  name: marsha-cloudfront-private-key-secret
+{% endif %}
+          volumes:
+            - name: marsha-configmap
+              configMap:
+                defaultMode: 420
+                name: marsha-app-{{ deployment_stamp }}
+{% if env_type in trashable_env_types %}
+            - name: marsha-v-media
+              persistentVolumeClaim:
+                claimName: marsha-pvc-media
+            - name: marsha-v-static
+              persistentVolumeClaim:
+                claimName: marsha-pvc-static
+{% endif %}
+{% if marsha_should_sign_requests %}
+            - name: marsha-cloudfront-private-key-secret
+              secret:
+                secretName: "{{ marsha_cloudfront_private_key_secret_name }}"
+{% endif %}
```

Comment:
I wouldn't mount static, media and cloudfront private key...

Reply:
I'm not sure it will work without cloudfront private key